pkg/qemu/qemu.go

```diff
@@ -1164,3 +1202,40 @@ func getFirmware(qemuExe string, arch limayaml.Arch) (string, error) {
 	qemuArch := strings.TrimPrefix(filepath.Base(qemuExe), "qemu-system-")
 	return "", fmt.Errorf("could not find firmware for %q (hint: try copying the \"edk-%s-code.fd\" firmware to $HOME/.local/share/qemu/)", arch, qemuArch)
 }
+
+func getFirmwareVars(qemuExe string, arch limayaml.Arch) (string, error) {
+	var targetArch string
+	switch arch {
+	case limayaml.X8664:
+		targetArch = "i386"
```

Comment:
?

Reply:
x86_64 and i386 use the same layout for vars, so, there is no x86_64 version in the distribution. One needs to concatenate i386 to x86_64 code. This is the legacy of x86_64 being derivative of i386. Probably this deserves explanation with a comment.